news/_posts/core-weekly/2018-10-12-coreweekly-week-40-2018.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 40 of 2018"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2018-10-12 16:45:00
+authors: [ AntoineThomas ]
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase during the last week, from Monday 17th of August to Sunday 23rd of September 2018.
+
+![Core Weekly banner](/assets/images/2017/04/core_weekly_banner.jpg)
+
+
+## General messages
+
+Dear Developers, there are a lot of things to tell this week:
+
+* [1.6.1.22 is out to fix a bug introduced in 1.6.1.21](http://build.prestashop.com/news/prestashop-1-6-1-22-maintenance-release/). If you have a shop in 1.6.1.21, this is mandatory to update. 
+* It's official, the [end of life of PrestaShop 1.6.1.x is now in June 2019](https://www.prestashop.com/en/blog/Maintenance-extension-PrestaShop-1-6)
+* We had a lot of issues with our APIs this week, and we are very sorry. Be assured that our top priority is to make sure that every PrestaShop merchant is not disturbed anymore with this kind of issue, especially for the upcoming christmas season.
+* Because of the other top priorities of this week, 1.7.5 beta is late. More news about it next week.
+
+
+## A quick update about GitHub issues
+
+Last week, [65 new issues](https://github.com/PrestaShop/PrestaShop/issues?utf8=%E2%9C%93&q=is:issue+created:2018-10-01..2018-10-07)  issues have been opened in PrestaShop's core repository, and [20 fixed issues](https://github.com/PrestaShop/PrestaShop/issues?utf8=%E2%9C%93&q=is:issue+label:fixed+closed:2018-10-01..2018-10-07) have been closed.
```

Comment:
there is an extra space before "issues have been opened", does it change the rendering?